src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json

```diff
@@ -9,9 +32,13 @@
                 "headline": "Current week",
                 "content": "The exercises of the current week will be displayed on top of the exercise list."
             },
+            "currentExercise": {
+                "headline": "Current Exercise",
+                "content": "In this tutorial we will have a look at the 'Hello World' exercise."
+            },
             "exerciseType": {
                 "headline": "Exercise type",
-                "content": "The exercise type can be distinguished by the exercise icon."
+                "content": "The exercise type can be distinguished by the exercise icon. This example here is a programming exercise."
```

Comment:
`This example here is a programming exercise.` -> `In this example it is a programming exercise` or simply `Here it is a programming exercise`